src/main/java/org/spongepowered/api/event/TristateResult.java

```diff
@@ -60,19 +60,17 @@
     }
 
     /**
-     * Returns whether the
-     * {@link TristateResult.Result#ALLOW} can be
-     * used.
+     * @return Whether the {@link TristateResult.Result#ALLOW} can be used
```

Comment:
Interesting, checkstyle actually doesn't catch that one. For reference, 
the current (complete) log:

```
[ant:checkstyle] 
src/main/java/org/spongepowered/api/GameRegistry.java:547:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/GameRegistry.java:549:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/GameRegistry.java:551:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/GameRegistry.java:553:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/GameRegistry.java:555:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/GameRegistry.java:557:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/GameRegistry.java:559:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/GameRegistry.java:561:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/GameRegistry.java:563:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/GameRegistry.java:565:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/GameRegistry.java:567:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/block/BlockTransaction.java:51:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/block/BlockTransaction.java:55:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/manipulator/ImmutableDataManipulatorBuilder.java:60:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/merge/MergeFunction.java:92:9: 
warning: Redundant 'final' modifier.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/DirectionRelativePropertyHolder.java:34:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/LocationBasePropertyHolder.java:128:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/PropertyStore.java:66:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/block/GravityAffectedProperty.java:38:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/block/GravityAffectedProperty.java:42:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/block/HeldItemProperty.java:35:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/block/HeldItemProperty.java:39:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/block/MatterProperty.java:35:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/block/MatterProperty.java:39:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/block/ReplaceableProperty.java:36:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/block/ReplaceableProperty.java:40:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/block/StatisticsTrackedProperty.java:32:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/block/StatisticsTrackedProperty.java:36:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/block/UnbreakableProperty.java:31:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/block/UnbreakableProperty.java:35:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/entity/EyeHeightProperty.java:31:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/entity/EyeHeightProperty.java:35:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/entity/EyeLocationProperty.java:35:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/property/entity/EyeLocationProperty.java:39:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/data/value/mutable/CompositeValueStore.java:254:9: 
warning: Redundant 'final' modifier.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/entity/Transform.java:68:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/entity/Transform.java:72:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/entity/Transform.java:76:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/entity/Transform.java:80:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/entity/Transform.java:84:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/entity/Transform.java:88:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/Cause.java:98:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/damage/source/BlockDamageSourceBuilder.java:52:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/damage/source/BlockDamageSourceBuilder.java:54:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/damage/source/DamageSourceBuilder.java:31:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/damage/source/DamageSourceBuilder.java:33:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/damage/source/DamageSourceBuilder.java:35:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/damage/source/DamageSourceBuilder.java:37:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/damage/source/DamageSourceBuilder.java:39:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/damage/source/DamageSourceBuilder.java:41:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/damage/source/DamageSourceBuilder.java:43:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/damage/source/EntityDamageSourceBuilder.java:50:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/damage/source/FallingBlockDamageSourceBuilder.java:54:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/damage/source/ProjectileDamageSourceBuilder.java:54:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/BlockHealingSource.java:34:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/BlockHealingSource.java:36:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/BlockHealingSourceBuilder.java:52:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/BlockHealingSourceBuilder.java:54:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/EntityHealingSource.java:32:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/EntityHealingSourceBuilder.java:51:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/HealingSource.java:45:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/HealingSourceBuilder.java:32:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/HealingSourceBuilder.java:34:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/HealingSourceBuilder.java:36:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/HealingSourceBuilder.java:38:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/HealingSourceBuilder.java:40:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/HealingSourceBuilder.java:42:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/HealingSourceBuilder.java:44:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/ProjectileHealingSource.java:36:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/health/source/ProjectileHealingSourceBuilder.java:55:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/BlockSpawnCause.java:31:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/BlockSpawnCauseBuilder.java:32:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/BreedingSpawnCause.java:36:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/BreedingSpawnCauseBuilder.java:33:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/BreedingSpawnCauseBuilder.java:35:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/EntitySpawnCause.java:31:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/EntitySpawnCauseBuilder.java:33:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/EntitySpawnCauseBuilder.java:35:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/MobSpawnerSpawnCause.java:31:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/MobSpawnerSpawnCauseBuilder.java:33:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/SpawnCause.java:39:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/SpawnCauseBuilder.java:30:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/SpawnCauseBuilder.java:32:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/spawn/WeatherSpawnCauseBuilder.java:32:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/cause/entity/teleport/EntityTeleportCause.java:31:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/event/world/ConstructPortalEvent.java:41:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/item/inventory/ItemStackTransaction.java:48:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/potion/PotionEffectBuilder.java:34:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/Functional.java:75:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/Functional.java:79:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/Tuple.java:39:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/Tuple.java:46:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/Tuple.java:51:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/Tuple.java:55:5: warning: 
Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/blockray/BlockRay.java:459:20: 
warning: Expected @param tag for '<E>'.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/blockray/BlockRay.java:471:20: 
warning: Expected @param tag for '<E>'.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/blockray/BlockRay.java:634:20: 
warning: Expected @param tag for '<E>'.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/blockray/BlockRay.java:647:20: 
warning: Expected @param tag for '<E>'.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/blockray/BlockRay.java:657:20: 
warning: Expected @param tag for '<E>'.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/blockray/BlockRay.java:675:20: 
warning: Expected @param tag for '<E>'.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/blockray/BlockRayHit.java:38: 
warning: Javadoc comment at column 14 has parse error. Missed HTML close 
tag 'E'. Sometimes it means that close tag missed for one of previous 
tags.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/command/args/PatternMatchingCommandElement.java:61:75: 
warning: Reference to instance variable 'getValue' needs "this.".
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/event/factory/ClassGenerator.java:217:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/event/factory/ClassGenerator.java:259:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/util/event/factory/ClassGenerator.java:556:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/TeleporterAgent.java:32:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/TeleporterAgent.java:34:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/TeleporterAgent.java:36:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/TeleporterAgent.java:38:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/TeleporterAgent.java:40:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/TeleporterAgent.java:42:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/TeleporterAgent.java:44:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/TeleporterAgent.java:46:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/TeleporterAgent.java:48:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/WorldBuilder.java:170:5: 
warning: Missing a Javadoc comment.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/extent/LocationCompositeValueStore.java:426:13: 
warning: Redundant 'final' modifier.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/extent/LocationCompositeValueStore.java:587:9: 
warning: Redundant 'final' modifier.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/extent/LocationCompositeValueStore.java:609:9: 
warning: Redundant 'final' modifier.
[ant:checkstyle] 
src/main/java/org/spongepowered/api/world/extent/LocationCompositeValueStore.java:633:9: 
warning: Redundant 'final' modifier.
```